src/mlpack/methods/cf/cf_main.cpp

```diff
@@ -142,17 +161,139 @@ void ComputeRecommendations(CFModel* cf,
 
     Log::Info << "Generating recommendations for " << users.n_elem << " users."
         << endl;
-    cf->GetRecommendations(numRecs, recommendations, users.row(0).t());
+
+    //  Making All the possible paths
+    if (ns_algo == "Cosine")
+    {
+        if (iw_algo == "Average")
+        {
+            cf->GetRecommendations<CosineSearch,
+              AverageInterpolation>(numRecs, recommendations,
```

Comment:
To wrap a line, please use two tabs (4 spaces).
https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping